### PR TITLE
Improve display of last failure

### PR DESCRIPTION
**Pull request by kamaradclimber**
It will allow to hint user if task failure is recent (using red color to
indicate there might be something to do), a bit old (using yellow to
indicate it might the info they are looking for) or very old (using
green color to show it is likely ok despite being a failure).

Also precise some field naming

Change-Id: I66e827d1a3796de6af658c55a9231c8fdf659126

### CODE REVIEW

**Comment by kamaradclimber** [comment]
Screenshots

image
image

File: src/js/components/AppDebugInfoComponent.jsx [hunk 1]
```diff
@@ -70,23 +71,36 @@ var AppDebugInfoComponent = React.createClass({
...
         <dt>Message</dt>
         {invalidateValue(lastTaskFailure.message)}
         <dt>Host</dt>
         {invalidateValue(lastTaskFailure.host)}
-        <dt>Timestamp</dt>
+        <dt>Failure timestamp</dt>
         <dd>
-          <span>{timestamp}</span> ({timeStampText})
+          <span>{timestamp}</span>
+           (<span className={timestampClassSet}>{timeStampText}</span>)
```
Comment (warrenseine) [inline src/js/components/AppDebugInfoComponent.jsx:101]: You're missing a space before "(". It's visible in the screenshot.

To fix it, write it on the same line or use a `&nbsp;`.

**Review by warrenseine — CHANGES_REQUESTED** [review]
Good idea!

File: src/js/components/AppDebugInfoComponent.jsx (hunk 1 above)
Comment (kamaradclimber) [reply src/js/components/AppDebugInfoComponent.jsx:101]: done

**Review by Lqp1 — APPROVED** [review]

**Pull request merged**